reference/tugboat-configuration/index.md

```diff
@@ -34,11 +42,71 @@ the service:
 | [https](#https)                  | Boolean | Whether the service should be available via HTTPS         |
 | [https_redirect](#httpsredirect) | Boolean | Whether HTTP requests should be redirected to HTTPS       |
 | [domain](#domain)                | String  | A custom domain for Tugboat to generate URLs with         |
-| [subpath](#subpath)              | Boolean | Whether subpath URLs should be generated for this service |
 | [subpath_map](#subpathmap)       | Boolean | Whether to map the generated subpath to "/"               |
 
 ---
 
+## `alias_type`
+
+* **Type:** String
+* **Default:** `default`
+* **Required:** No
+
+What type of URL aliases to generate for the service. Valid options are
+`default` or `domain`. Alias URLs are generated in addition to the normal
+Service URLs.
+
+#### `default`
+
+When `alias_type` is set to `default`, the alias URLs are constructed by
+substituting the preview name in the Service URL with the values of `aliases`.
+The values of `aliases` are sanitized based on the value of
+[`subpath`](#subpath).
+
+If `subpath` is `false`, alias values are sanitized to create a valid host name,
+and is truncated to 30 characters, to make the host name a maximum of 63
+characters. If `subpath` is `true`, the alias values are URL encoded.
+
+If `aliases` is set to `['foo', 'bar']`, alias URLs look like the following
+
+* https://foo-4vdrhxvyddvr5tne7zcr4y72vzowqohj.tugboat.qa
+* https://bar-4vdrhxvyddvr5tne7zcr4y72vzowqohj.tugboat.qa
+
+or
+
+* https://previews.tugboat.qa/foo-4vdrhxvyddvr5tne7zcr4y72vzowqohj/
+* https://previews.tugboat.qa/bar-4vdrhxvyddvr5tne7zcr4y72vzowqohj/
+
+#### `domain`
+
+When `alias_type` is set to `domain`, the alias URLs are constructing by
+substituting the domain part of the Service URL with the values of `aliases`.
+The alias values are sanitized to create a valid domain name. If `aliases` is
+set to `['foo.com', 'bar.com']`, alias URLs look like the following
+
+* https://pr123-4vdrhxvyddvr5tne7zcr4y72vzowqohj.foo.com
+* https://pr123-4vdrhxvyddvr5tne7zcr4y72vzowqohj.bar.com
+
+or
+
+* https://previews.foo.com/pr123-4vdrhxvyddvr5tne7zcr4y72vzowqohj/
```

Comment:
It actually doesn't prepend previews to the domain. It would just be `https://foo.com/pr123-4vdrhxvyddvr5tne7zcr4y72vzowqohj/